src/Module/Config/Step/OverrideConfigFromCLI.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Couscous\Module\Config\Step;
+
+use Couscous\Model\Project;
+use Psr\Log\LoggerInterface;
+
+/**
+ * Override config variables when specified using --config option.
+ *
+ * @author D.J. Marcolesco <dj.marcolesco@gmail.com>
+ */
+class OverrideConfigFromCLI implements \Couscous\Step
+{
+    /**
+     * @var LoggerInterface
+     */
+    private $logger;
+
+    public function __construct(LoggerInterface $logger)
+    {
+        $this->logger = $logger;
+    }
+
+    public function __invoke(Project $project)
+    {
+        if ($project->metadata['tempConfigRaw']) {
```

Comment:
Minor detail but you can [return early](http://blog.humphd.org/vocamus-1421/) to avoid 1 level of indentation